stock_inventory_valuation_report/wizard/stock_quantity_history_view.xml

```diff
@@ -3,7 +3,7 @@
     <record id="view_stock_quantity_history_inherit" model="ir.ui.view">
         <field name="name">view_stock_quantity_history_inherit</field>
         <field name="model">stock.quantity.history</field>
-        <field name="inherit_id" ref="stock_account.view_stock_quantity_history"/>
+        <field name="inherit_id" ref="stock.view_stock_quantity_history"/>
```

Comment:
@bodedra `stock_account.view_stock_quantity_history` and `stock.view_stock_quantity_history` are different views. 
- `stock_account.view_stock_quantity_history` => Valuation Report
- `stock.view_stock_quantity_history` => Inventory Report

This module adds report buttons on Inventory Valuation that view is the Valuation Report. So this code should use `stock_account.view_stock_quantity_history`.